system/modules/calendar/templates/calendar/cal_default.html5

```diff
@@ -19,8 +19,9 @@
         <td class="<?php echo $day['class']; ?>">
           <div class="header"><?php echo $day['label']; ?></div>
           <?php foreach ($day['events'] as $event): ?>
-            <div class="event cal_<?php echo $event['parent']; ?><?php echo $event['class']; ?>">
-              <a href="<?php echo $event['href']; ?>" title="<?php echo $event['title']; ?> (<?php if ($event['day']): echo $event['day']; ?>, <?php endif; echo $event['date']; if ($event['time']): ?>, <?php echo $event['time']; endif; ?>)"<?php echo $event['target']; ?>><?php echo $event['link']; ?></a>
+            <div class="event cal_<?php echo $event['parent']; ?><?php echo $event['class']; ?>" itemscope itemtype="http://schema.org/Event">
+              <a href="<?php echo $event['href']; ?>" title="<?php echo $event['title']; ?> (<?php if ($event['day']): echo $event['day']; ?>, <?php endif; echo $event['date']; if ($event['time']): ?>, <?php echo $event['time']; endif; ?>)"<?php echo $event['target']; ?> itemprop="url"><span itemprop="name"><?php echo $event['link']; ?></span></a>
```

Comment:
`<span itemprop="name">` is inside the URL
The markup has to look like this
```
<div itemscope itemtype="http://schema.org/Event">
  <a href="#" itemprop="url">
    <span itemprop="name">Event Name</span>
    <time itemprop="startDate" content=""></time>
  </a>
</div>
```

I've added an extra span because `itemprop="name"` has to be a child of `itemtype="http://schema.org/Event"` and can not be added to the `<a>` tag. Otherwise the name would be the content of the anchor